readme.md

```diff
@@ -505,6 +505,9 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 ## Other Awesome Lists
 
 - [awesome-git-addons](https://github.com/stevemao/awesome-git-addons) - A curated list of addons that extends git cli.
+- [awesome-macos-command-line](https://github.com/herrbischoff/awesome-macos-command-line) - A curated list of shell commands and tools specific to macOS.
+- [awesome-command-line-apps](https://github.com/herrbischoff/awesome-command-line-apps) – An alternative awesome list of cli apps.
+- [awesome-shell](https://github.com/alebcay/awesome-shell) – A curated list of awesome command-line utilities and frameworks.
```

Comment:
Remove the "A curated list of awesome" from the front, if you are on an awesome list, you know what they are.

Reply:
Done ✅